src/ZfcRbac/Initializer/AuthorizationServiceInitializer.php

```diff
@@ -32,17 +32,22 @@
 class AuthorizationServiceInitializer implements InitializerInterface
 {
     /**
-     * @see \Zend\ServiceManager\InitializerInterface::initialize()
+     * @param ContainerInterface $container
+     * @param object $instance
      */
-    public function initialize($instance, ServiceLocatorInterface $serviceLocator)
+    public function __invoke(ContainerInterface $container, $instance)
     {
         if ($instance instanceof AuthorizationServiceAwareInterface) {
-            if ($serviceLocator instanceof AbstractPluginManager) {
-                $serviceLocator = $serviceLocator->getServiceLocator();
-            }
-            
-            $authorizationService = $serviceLocator->get('ZfcRbac\Service\AuthorizationService');
+            $authorizationService = $container->get('ZfcRbac\Service\AuthorizationService');
             $instance->setAuthorizationService($authorizationService);
         }
     }
+
+    /**
+     * @see \Zend\ServiceManager\InitializerInterface::initialize()
+     */
+    public function initialize($instance, ServiceLocatorInterface $serviceLocator)
+    {
+        $this($serviceLocator, $instance);
```

Comment:
should check if this is `AbstractPluginManager` and use `$serviceLocator->getServiceLocator()`